R/repr_matrix_df.r

```diff
@@ -50,7 +60,7 @@ ellip.limit.arr <- function(
 		# data.tables can't be indexed by column number, unless you provide the
 		# with=FALSE parameter. To avoid the hassle, just convert to a normal table.
 		if (inherits(a, 'data.table'))
-			a <- as.data.frame(x)
```

Comment:
This was a bug? Otherwise, this looks strange to change that without a change which introduces a above...